ni_measurementlink_service/_drivers/_nidcpower.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import annotations
+
+from typing import Any, Dict, Optional
+
+import nidcpower
+
+from ni_measurementlink_service import session_management  # circular import
```

Comment:
I will probably move `SessionInformation` into `_sessiontypes.py` in a separate PR to break this cycle.

Reply:
https://github.com/ni/measurementlink-python/pull/433